src/context.rs

```diff
@@ -75,9 +79,23 @@ impl PySessionContext {
         repartition_windows: bool,
         parquet_pruning: bool,
         target_partitions: Option<usize>,
-        // TODO: config_options
+        config_options: Option<HashMap<String, String>>,
```

Comment:
Does this mean we have to pass the stringized versions of rust types as config values? E.g. `{"batch_size": "4096"}` (instead of literal `4096`, or `"true"` instead of `True`). I guess it might be an improvement later (just to keep in mind).

Reply:
Yes, for now .. I wasn't sure how to do it in a type-safe way with Python. It would be good to look at improving this in a follow on PR.

I tried using BuiltInConfig but the data is private so I could not access it from this project.